apps/web/pages/reschedule/[uid].tsx

```diff
@@ -76,7 +77,9 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 
   return {
     redirect: {
-      destination: `/${eventPage}?${destinationUrl.toString()}`,
+      destination: `/${eventPage}?${destinationUrl.toString()}${
+        eventType.seatsPerTimeSlot ? "&bookingUid=null" : ""
+      }`,
```

Comment:
Noticed that you would have to double click on a timeslot when `bookingUid` was not already in the URL params. Also setting to null as "" was not valid and messing with getting the URL params.

Reply:
Why do we need to set `&bookingUid=null` ? Is skipping it no enough?

Reply:
@alannnc we don't need this now after you found the fix?